ApplyToTheIndustry/Assets/ScriptableObjects/Job Postings/GameProducerQuad.asset

```diff
@@ -0,0 +1,26 @@
+%YAML 1.1
+%TAG !u! tag:unity3d.com,2011:
+--- !u!114 &11400000
+MonoBehaviour:
+  m_ObjectHideFlags: 0
+  m_CorrespondingSourceObject: {fileID: 0}
+  m_PrefabInstance: {fileID: 0}
+  m_PrefabAsset: {fileID: 0}
+  m_GameObject: {fileID: 0}
+  m_Enabled: 1
+  m_EditorHideFlags: 0
+  m_Script: {fileID: 11500000, guid: d8cfffae36be18b40b59e8e84d47a6ad, type: 3}
+  m_Name: GameProducerQuad
+  m_EditorClassIdentifier: 
+  company: {fileID: 11400000, guid: 9cfc23b03c89faa4fa9248e30f7d815a, type: 2}
+  positionName: Game Producer
+  jobSummaryLines:
+  - We are urgently looking for a game producer!
+  - 'Skills required:'
+  jobReqLines:
+  - '>5-10 years of working in the gaming industry'
```

Comment:
"years of working" -> "years experience working"
"gaming industry" -> "games industry"
(**gaming** industry refers primarily to gambling, casinos, etc.)